syntax = "proto2";

package harmonyServer;

message StopModel {
  required string experiment_id = 1;
}
